### ISSUE TITLE
Some systems failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [melisgl/mgl-mat#5]
Building with SBCL 2.2.7.28-02bc916fd / ASDF 3.3.5 for quicklisp dist creation.

Trying to build commit id c78ccbcc76df3c32004d18084d885c7cbe3618bc

*mgl-mat* fails to build with the following error:

```
; caught WARNING:
;   Error during compiler-macroexpansion of (CFFI:FOREIGN-TYPE-SIZE (QUOTE #)). Use *BREAK-ON-SIGNALS* to intercept.
;    Unknown CFFI type (:STRUCT CURAND-STATE-XORWOW)
...
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING {10016C8003}>: COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "mgl-mat" "src" "curand">

```

*mgl-mat/test* fails to build because of a failure in  _mgl-mat_.

[Full log here](http://report.quicklisp.org/2022-11-18/failure-report/mgl-mat.html)

**Comment by quicklisp** [comment]
See also https://github.com/cffi/cffi/issues/346#issuecomment-1320005306

**Comment by melisgl** [comment]
Thank you. Pushed 2095301, which fixes it here with the cffi 8629618.
